cmd/gencerts_test.go

```diff
@@ -0,0 +1,89 @@
+package cmd_test
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/OpenBazaar/openbazaar-go/cmd"
+)
+
+func TestGenCertsGenericDefaults(t *testing.T) {
+
+	if dataDir, err := ioutil.TempDir("", "gencerts_test"); err != nil {
+		t.Fatal(err)
+	} else {
```

Comment:
You don't need the `else` case here. `t.Fatal` will exit the function as soon as it finishes.